### PR TITLE
faucet should use the current deployment

### PR DESCRIPTION
**Pull request by vujevits**

### CODE REVIEW

File: packages/app/src/data/stakers.tsx [hunk 1]
```diff
@@ -51,6 +38,19 @@ const stakers: Record<string, Staker> = {
...
+    description:
+      "Matic Network brings massive scale to Ethereum using an adapted version of Plasma with PoS based side chains. The Matic team a decentralized platform using an adapted version of Plasma framework that provides a solution for faster and extremely low-cost transactions with finality on the main chain. The system ensures liveliness using PoS checkpoints which are pushed to the Ethereum mainchain. This enables a single Matic sidechain to theoretically achieve 2 16 transactions per block, and possibly millions of transactions on multiple chains in the future.",
+    stakerAddress: "",
+    available: false,
+    apy: 14.8,
+    logo: "matic.svg",
+    bwLogo: "LPT.svg",
+    bwTenderLogo: "tenderLPT.svg",
+    symbol: "MATIC",
+  },
```
Comment (kyriediculous) [inline packages/app/src/data/stakers.tsx:53]: Let's remove or comment this one out for now

File: packages/app/src/components/faucet/index.tsx [hunk 2]
```diff
@@ -13,11 +14,17 @@ const Faucet: FC<props> = ({ symbol }) => {
   const handleClose = () => setShow(false);
   const handleShow = () => setShow(true);
 
-  const { send } = useContractFunction(contracts.livepeer.faucet, "request");
+  const location = useLocation();
+  const name = location.pathname.split("/")[2];
 
+  console.log(contracts[name]);
```
Comment (kyriediculous) [inline packages/app/src/components/faucet/index.tsx:20]: Let's remove the console.log here

File: packages/app/src/components/faucet/index.tsx [hunk 3]
```diff
@@ -13,11 +14,17 @@ const Faucet: FC<props> = ({ symbol }) => {
...
+  const { send } = useContractFunction(contracts[name].faucet, "request");
+
+  // TODO why is this needed
   const requestTokens = () => {
     send();
   };
+
```
Comment (kyriediculous) [inline packages/app/src/components/faucet/index.tsx:27]:
```suggestion
  const { send: requestTokens } = useContractFunction(contracts[name].faucet, "request");
```

**Review by kyriediculous — CHANGES_REQUESTED** [review]

File: packages/app/src/components/faucet/index.tsx (hunk 3 above)
Comment (vujevits) [reply packages/app/src/components/faucet/index.tsx:27]: actually I tried that, and does not work, hence I added the TODO

**Review by kyriediculous — APPROVED** [review]

**Pull request merged**